src/helpers/polygon.ts

```diff
@@ -3,6 +3,13 @@ interface Bounds {
   sw: string
 }
 
+const pointRegexRule = (point: string) => /^(-?\d+(\.\d+)?),\s*(-?\d+(\.\d+)?)$/.test(point)
+
+export const isRequestBounds = (bounds: Bounds) => bounds?.ne
+    && bounds.sw
+    && pointRegexRule(bounds.ne)
+    && pointRegexRule(bounds.sw)
+
```

Comment:
untuk apaniii man wkwkw kepoo

Reply:
<img width="676" alt="image" src="https://user-images.githubusercontent.com/41193120/165430638-021531ff-2ef1-4233-8d13-34afe818c6f6.png">
buat cek ini van, harus sesuai format

Reply:
ada alasannya mang namain sw/ne? @ayocodingit atau memang dari polygon nya?

Reply:
dari sini mang 
https://aiocollective.com/blog/getbounds-in-google-maps-api-v3/
sw: south west
ne: north west

Reply:
hooo mantep mantep 😸